requirements/travis.in

```diff
@@ -0,0 +1,7 @@
+# Requirements for running tests in Travis
+-c constraints.txt
+
+-r test.txt
```

Comment:
This is not need here

Reply:
I added it on purpose since tests were failing because they could not fin pep8 https://travis-ci.org/github/edx/xblock-lti-consumer/jobs/682161790
Should I just add pep8 to the travis.in file ?

Reply:
I just realized that this is not using tox. So I guess that the thing that we should do is migrate the tests to tox. We can do this in this PR or in another PR in the future, so I think that we can keep this until we make that change.